test/scripts/install-deb.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# This script installs the required deb packages to bootstrap a Kubernetes cluster
+# It is referenced from ../deb/Vagrantfile
+
+# Package configs
+KUBERNETES_VERSION=v1.30
+PROJECT_PATH=prerelease:/main
```

Comment:
Do we want to source this from another file and include it here as well as in install-rpm.sh?

Reply:
It would be nice, but I don't think `vagrant` tracks script dependencies.

Reply:
Implemented in [commit](https://github.com/cri-o/packaging/pull/112/commits/831c9caa7dd23792d65a6275977921579b607a63).